.github/workflows/main.yml

```diff
@@ -1,10 +1,13 @@
 name: CI
 
-on: [push, pull_request]
+on: push
```

Comment:
not sure if thats ok?

Reply:
If it runs for forks, too - it's fine :)

Reply:
top